distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Match_2.enso

```diff
@@ -21,18 +28,15 @@ type Match_2
     start : Integer -> Integer
     start self n = self.internal_regex_result.getStart n
 
-    ## PRIVATE
-       Returns the start character of group n
+    ## Returns the start character of group n
```

Comment:
```suggestion
    ## Returns the final UTF16 character index of group n.
```

Reply:
Using:
    ## Returns the end UTF16 character index, plus one, of group n.